test/baas_helper.dart

```diff
@@ -97,38 +86,41 @@ class BaasHelper {
     }
   }
 
-  static bool shouldRunBaasTests(Map<String, String?> args) {
-    return args[argBaasaasApiKey] != null || args[argBaasUrl] != null;
+  static bool get shouldRunBaasTests {
+    return Env.baasUrl.isDefined || Env.baasAasApiKey.isDefined;
   }
 
   BaasHelper._(this._baasClient);
 
-  static Future<BaasClient?> _setupClient(Map<String, String?> args) async {
-    var baasUrl = args[argBaasUrl];
-    final differentiator = args[argDifferentiator] ?? 'local';
+  static Future<BaasClient?> _setupClient() async {
+    var baasUrl = Env.baasUrl();
+    final cluster = Env.baasCluster();
+    final apiKey = Env.baasApiKey();
+    final privateApiKey = Env.baasPrivateApiKey();
+    final projectId = Env.baasProjectId();
+    final differentiator = Env.differentiator();
+
     if (baasUrl == null) {
-      final baasaasApiKey = args[argBaasaasApiKey];
-      if (baasaasApiKey != null) {
-        if (args[argBaasCluster] != null) {
-          throw "$argBaasaasApiKey can't be combined with $argBaasCluster";
+      final baasAasApiKey = Env.baasAasApiKey();
+      if (baasAasApiKey != null) {
+        if (cluster != null) {
+          throw "BAAS_BAASAAS_API_KEY can't be combined with BAAS_CLUSTER";
         }
 
-        (baasUrl, _) = await BaasClient.retry(() => BaasClient.getOrDeployContainer(baasaasApiKey, differentiator));
+        (baasUrl, _) = await BaasClient.retry(() => BaasClient.getOrDeployContainer(
+              baasAasApiKey,
+              differentiator!,
```

Comment:
Is this safe? As far as I can tell, there's nothing that guarantees that a differentiator is supplied.

Reply:
You are right. Will fix and use 'local' as default.